developer-docs-site/docs/standards/fungible-asset.md

```diff
@@ -0,0 +1,407 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FAs) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible units, which can represent different assets, such as currency, shares, material in games, or many other types of assets. Furthermore, FAs can be used to build decentralized applications that require a token economy, such as decentralized exchanges or gaming platforms.
+
+The [fungible asset module](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move) provides a standard, type-safe framework for defining FAs within the Aptos Move ecosystem.
+
+In this standard, FAs are stored in `Object<FungibleStore>` that has a specific amount of units. FAs are units that are interchangeable with others of the same metadata. The standard is built upon object model so all the resources defined here are included in object resource group and stored inside objects. FAs have a metadata object define that include information such as name, symbol and decimals. They can be stored in objects that contain a FungibleStore resource. These store objects can be freely created and FAs can be moved, split, combined between them easily. The standard also supports minting new units and burning existing units with appropriate controls.
+
+The different objects involved - `Object<Metadata>` and `Object<FungibleStore>` objects, and their relationships to accounts are shown in the diagram below:
+<div style={{textAlign: "center"}}>
+<ThemedImage
+alt="fungible asset architecture"
+sources={{
```

Comment:
I wonder if we can keep primary/secondary concepts out of this diagram. This should only demonstrate the metadata and store objects and their relationship to the creator and hodler accounts

Reply:
I think that's kind of easy for follow but without secondary stores, it is hard for ppl to realize primary store is not the only one an owner can have but a special PDA-store. Not to mention we introduce `secondary` in this doc but it's not in the diagram. Would that be more perplexing?